tests/catch/hipTestMain/config/config_amd_windows.json

```diff
@@ -66,7 +66,9 @@
 	   "Unit_hipHostMalloc_Coherent",
 	   "Unit_hipHostMalloc_Default",
 	   "Unit_hipStreamCreate_MultistreamBasicFunctionalities",
-	   "Unit_hipEventIpc"
+	   "Unit_hipEventIpc",
+	   "Unit_hipMemGetInfo_Negative",
+	   "# The prior tests relates to ticket EXSWCPHIPT-61"
```

Comment:
Write that info inside the test as a comment or something

Reply:
Changes to the json files have been removed, added ```HIP_SKIP_TEST``` inside test cases with relevant information